Automattic/jetpack
# SSO: remove legacy codebase in favor of code in Connection package

Fixes https://github.com/Automattic/vulcan/issues/385

## Proposed changes:

This codebase was deprecated in #37153, it's now time to remove it.

### Other information:


## Jetpack product discussion

* https://github.com/Automattic/vulcan/issues/251

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

The existing feature should continue to work as expected, and you should not see any deprecation warnings when tailing your error log.

* Go to Jetpack > Settings > Security
* Enable SSO
* Log out and back in using SSO.
